src/app/tide/PeptideDiskLoader.h

```diff
@@ -0,0 +1,107 @@
+#include <deque>
+#include <list>
+#include <unordered_set>
+#include "peptides.pb.h"
+#include "peptide.h"
+#include "theoretical_peak_set.h"
+#include "fifo_alloc.h"
+#include "spectrum_collection.h"
+#include "io/OutputFiles.h"
+#include "boost/thread/shared_mutex.hpp"
+#include "boost/thread/shared_lock_guard.hpp"
+
+#ifndef ACTIVE_PEPTIDE_QUEUE_H
+#define ACTIVE_PEPTIDE_QUEUE_H
+
```

Comment:
Add a comment section, please to indicate that this used to be ActivePeptideWindow. Please also explain the logic here, what and how this code is supposed to work.